src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/SomaticGenotypingEngine.java

```diff
@@ -122,45 +128,36 @@ public CalledHaplotypes callMutations (
                 readAlleleLikelihoods.contaminationDownsampling(MTAC.getSampleContamination());
             }
 
```

Comment:
There are TODOs above.  Should those be addressed or will they be addressed later?